Formula/nvm.rb

```diff
@@ -39,6 +39,6 @@ def caveats; <<-EOS.undent
     output = pipe_output("NODE_VERSION=homebrewtest #{prefix}/nvm-exec 2>&1")
     assert_no_match /No such file or directory/, output
     assert_no_match /nvm: command not found/, output
-    assert_match "N/A: version \"homebrewtest\" is not yet installed", output
+    assert_match /N\/A: version "N\/A" is not yet installed/, output
```

Comment:
`assert_match` is happy doing a partial assert though, even with `""`. `assert_equal` is the one that wants the whole cake, I believe.

`assert_no_match` won't take a string, but `assert_match` will. We prefer to use a string when possible because it means we don't have to escape slashes or full stops, which helps with readability at a glance.